ironfish/src/memPool/memPool.ts

```diff
@@ -110,6 +110,16 @@ export class MemPool {
     return true
   }
 
+  async getEstimatedFee(): Promise<{ fee: bigint;}> {
+    let memPoolEntry = this.queue.peek();
+    if (memPoolEntry == undefined){
+      return {fee: BigInt(0)}
+    }
+    else{
+      return { fee: memPoolEntry.fee }
+    }
```

Comment:
This is more of a preference, but in these cases we usually don't use an else, just have the main use case as the last statement.